test/commands.js

```diff
@@ -633,7 +662,7 @@ async function runMVMSendTokensCommand(command, state) {
       state.MVMBurnedTokens = state.MVMBurnedTokens.plus(lifWei);
       state.returnedWeiForBurnedTokens = state.returnedWeiForBurnedTokens.plus(tokensCost);
       state.balances[command.from] = getBalance(state, command.from).minus(lifWei);
-      state.MVMMaxClaimableWei = getMMMaxClaimableWei(state);
+      state.MVMMaxClaimableWei = getMvmMaxClaimableWei(state);
```

Comment:
here it shouldnt be `getMVMMaxClaimableWei` to follow the use of MVM on uppercase on every variable ?

Reply:
yeah, I changed it because it was too many `M`s together :), I thought this way had better readability. Not sure whether should we favor consistency or readability :)

what do you think?

Reply:
I think it still readable, I would prefer to follow the consistency in this case.